src/macros.jl

```diff
@@ -577,6 +577,32 @@ macro subset(x, args...)
     esc(subset_helper(x, args...))
 end
 
+function rsubset_helper(x, args...)
+    exprs, outer_flags = create_args_vector(args...)
+    if outer_flags[Symbol("@byrow")][] == true
+        throw(ArgumentError("Redundant @byrow calls"))
+    end
+
+    outer_flags[Symbol("@byrow")][] = true
+
+    t = (fun_to_vec(ex; no_dest=true, outer_flags=outer_flags) for ex in exprs)
+    quote
+        $subset($x, $(t...); skipmissing=true)
+    end
+end
+
+
+"""
+    @rsubset(d, i...)
+
+Row-wise version of `@subset`, i.e. all opertaions use `@byrow` by
+default. See `? @subset` for details.
```

Comment:
```suggestion
Row-wise version of `@subset`, i.e. all operations use `@byrow` by
default. See [`@subset`](@ref) for details.
```

Maybe worth mentioning that it's equivalent to `@subset d @byrow begin... end` (if you keep support for that syntax)?

Same for similar docstrings.

Reply:
Need to apply the same fix to other docstrings.

Reply:
Fixed.